crates/pipeline-manager/src/runner/error.rs

```diff
@@ -273,6 +295,9 @@ impl Display for RunnerError {
             Self::RunnerDeploymentCheckError { error } => {
                 write!(f, "Deployment check failed: one or more deployment resources encountered a fatal error.\n\n{error}")
             }
+            Self::RunnerDeploymentSuspendComputeError { error } => {
+                write!(f, "Deployment suspend compute failed: {error}")
+            }
             Self::RunnerDeploymentShutdownError { error } => {
                 write!(f, "Deployment shutdown failed (will retry): {error}")
```

Comment:
No need to fix yet, but in general, the word Deployment is unclear and not ever used in our user-facing documentation. It would also be confusing given k8s Deployments. Stick to the term `Pipeline` here and in the instances before it.